files.go

```diff
@@ -0,0 +1,159 @@
+package dotfile
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+)
+
+const (
+	dotfileDir = ".dotfile"
+	dotfile    = "files.json"
+)
+
+type trackedFile struct {
+	Path    string   `json:"path"`
+	Commits []string `json:"commits"`
+}
+
+// Init sets up a file for dotfile to track.
+func Init(path string, fileName string) error {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return fmt.Errorf("%s not found", path)
```

Comment:
Use `%#v` instead of `%s`; it'll automatically wrap the string in quotes which makes debugging empty strings way easier

Reply:
Nice I didn't know about the # .